app/models/export/submissions/row.rb

```diff
@@ -60,11 +60,15 @@ def created_by
         submission.created_by.try(:name)
       end
 
-      # Fields that are nil for MVP
-      def supplier_approved_date; end
+      def supplier_approved_by
+        submission.submitted_by.try(:name)
```

Comment:
Not a blocker for this PR, but do you know about Ruby's safe navigation operator? http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/ more or less the same thing but baked into Ruby rather than ActiveSupport.

Reply:
I do! It seemed to make things more obscure for only one level of nil protection, but I can change it if it's better practice in the long term.

Reply:
We have no defined style guide for it and I have no strong feelings either way so happy to leave it as it is.